networking/network_observability/network-observability-overview.adoc

```diff
@@ -0,0 +1,47 @@
+:_content-type: ASSEMBLY
+[id='network-observability-overview']
+= About Network Observability
+include::_attributes/common-attributes.adoc[]
+:context: network-observability-overview
+
+toc::[]
+
+Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters.
+
+[id="dependencies-network-observability"]
+== Dependencies of Network Observability Operator
+
+The Network Observability Operator requires the following Operators:
+
+* Loki Operator: You must install Loki install before you install the Network Observability Operator. The Loki Operator is the backend that you can use to store all collected flows.
+* Grafana Operator: You can optionally install Grafana Operator for using custom dashboards and querying capabilities.
+* Kafka Operator: You can optionally install Kafka Operator. The Kafka Operator provides scalability, resiliency and high availability in the {product-title} cluster.
+
+[id="network-observability-operator"]
+== Network Observability Operator
+
+The Network Observability Operator deploys a monitoring pipeline to collect and enrich network flows. The NetObserv eBPF agent or CNI can create network flows. You can export these network flows in IPFIX format, such as OVN-Kubernetes.
+The Network Observability Operator provides dashboards, metrics, and keeps flows accessible in a log store, Grafana, and Loki. You can access these dashboards in the {product-title} console.
+
+[id="network-observability-dashboards"]
+== Network Observability metrics
+
+By using Network Observability metrics you can refine the stats by using the filters and display options. You can access the information for node, namespace, owner, pod, and service.
+
+[id="no-console-integration"]
+== {product-title} console integration
+
+{product-title} console integration offers topology view and Netflow tables.
+
+[id="network-observability-topology-views"]
+=== Network Observability topology views
+
+The topology view represents traffic between the {product-title} components as a graph. You can refine the stats by using the filters and display options. You can access the information for node, namespace, owner, pod, and service.
```

Comment:
Have three sections, one on metrics, another on topology, and a third for table.  
  
In the latest UI (see https://github.com/netobserv/network-observability-operator/blob/main/docs/assets/overview-dashboard.png), there are three tabs (not two).

Reply:
@stleerh there's already these three other PRs about it: https://github.com/openshift/openshift-docs/pull/53405 , https://github.com/openshift/openshift-docs/pull/53366 and https://github.com/openshift/openshift-docs/pull/53362
Maybe just put links to these sections